enterprise/server/remote_execution/runner/runner.go

```diff
@@ -1377,6 +1388,12 @@ func (p *pool) setLimits() {
 		p.maxRunnerCount, p.maxRunnerMemoryUsageBytes, p.maxRunnerDiskUsageBytes)
 }
 
+func taskSizesEqual(a, b *scpb.TaskSize) bool {
```

Comment:
Should we allow matching as long as the current requests are smaller than the limits (rather then just equal?)

Also, is there a way to compare against the actual limits rather than just what the previous task looked like? That way we could enable this for non-workflows too?

Reply:
Resolved offline to address this later.